protected/views/site/help.php

```diff
@@ -1,362 +1,929 @@
 <?php
```

Comment:
almost all content appears changed because of the autoformatting so I'll add clarifying comments